Bottom: 24dcf752f37dc408feca345f85553dfd4970546b
Top:    b6de38d9b5ab917380029a2205830aad565dbaa2
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 10:36:27 +0000

Refresh of try-from-first-principles-on

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 5e4e469..4b81334 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -26,7 +26,7 @@
 #define PERFECT_ROUND_CONTOURS
 
 #define REVERSED_PCB_CONTOURS 1 /* PCB Contours are reversed from the expected CCW for outer ordering - once the Y-coordinate flip is taken into account */
-//#undef REVERSED_PCB_CONTOURS
+#undef REVERSED_PCB_CONTOURS
 
 #ifdef REVERSED_PCB_CONTOURS
 #define COORD_TO_STEP_X(pcb, x) (COORD_TO_MM(                   (x)))
